### ISSUE TITLE
Visualize Error Distribution across folds

### ISSUE THREAD

**Issue by ngupta23** [pycaret/pycaret#1405]
**Is your feature request related to a problem? Please describe.**
Per this [comment on LinkedIn](https://www.linkedin.com/feed/update/urn:li:ugcPost:6814933350713741312?commentUrn=urn%3Ali%3Acomment%3A%28ugcPost%3A6814933350713741312%2C6814934502796779520%29&replyUrn=urn%3Ali%3Acomment%3A%28ugcPost%3A6814933350713741312%2C6815038277939273728%29) from @RamiKrispin

**Describe the solution you'd like**
If you are running a horse race between multiple models with backtesting, it is nice to visualize the error dist across the different partitions (beyond the error metrics table) and add the PI coverage rate on the testing partitions.

**Comment by ngupta23** [comment]
@ramiKrispin

Useful comments about distribution. I had added it to one of my previous packages - (cell 10 in [this notebook](https://github.com/AutoViML/Auto_TS/blob/master/example_notebooks/autots_univariate_example.ipynb)). Is this what you would like to see?

**Comment by RamiKrispin** [comment]
yes, bar plot or some other graphic that provides more context about the model performance beyond the error metrics. Here is an example I create in the past in the [TSstudio](https://ramikrispin.github.io/TSstudio/articles/forecasting.html#training-multiple-time-series-model-with-backtesting) package:
![image](https://user-images.githubusercontent.com/12760966/123833538-4d29c880-d8bb-11eb-8980-0b0ca31cc65f.png)

And animation:
![image](https://user-images.githubusercontent.com/12760966/123833618-66cb1000-d8bb-11eb-938b-475c80b20ddb.png)

**Comment by ngupta23** [comment]
Possible interfaces

```python
create_model("arima", plot = "cv") # Plots the results by fold overlaid with original time series
create_model("arima", plot = "metrics") # Plots the histogram and line plot by fold of metrics
```

**Comment by moezali1** [comment]
Closing as part of the 4.0 backlog cleanup — see #4177.

PyCaret 4.0 is a ground-up rewrite (OOP-only API, sklearn-native engine, Python 3.11+, sklearn ≥ 1.7) and almost all pre-4.0 issues either reference removed APIs (`setup()`, `plot_model`, `evaluate_model`, `interpret_model`, `automl`, the legacy time-series module, the Streamlit dashboard, …) or no longer reproduce on the new code path.

**If this still affects you in 4.0, please re-file as a new issue** with:

- the experiment constructor call (`Experiment(...)`),
- the operation that fails,
- the full traceback,
- your Python / sklearn / pycaret versions.

Resources for 4.0:

- Docs: https://pycaret.org/docs
- Migrate from 3.x: https://pycaret.org/docs/guides/migrating-from-3-x
- API reference: https://pycaret.org/reference
- FAQ: https://pycaret.org/docs/resources/faqs

Thanks for filing — and apologies for the long latency. The repo had drifted; we're starting clean.
